.github/workflows/pr_merged.yml

```diff
@@ -0,0 +1,36 @@
+name: Auto Comment on PR Merged
+
+on:
+  pull_request_target:
+    types: [closed]
```

Comment:
The type should not be `closed` ideally, I want the message to be sent only if it is merged successfully.